lib/maths/CBoostedTreeFactory.cc

```diff
@@ -1225,6 +1232,16 @@ CBoostedTreeFactory& CBoostedTreeFactory::eta(double eta) {
     return *this;
 }
 
+CBoostedTreeFactory& CBoostedTreeFactory::etaGrowthRatePerTree(double etaGrowthRatePerTree) {
+    if (etaGrowthRatePerTree < MIN_ETA) {
+        LOG_WARN(<< "Truncating supplied learning rate " << etaGrowthRatePerTree
```

Comment:
should we explicitly mention "growth rate" here to know which one it is?

Reply:
Corrected in [1bd2a41](https://github.com/elastic/ml-cpp/pull/1683/commits/1bd2a414085496269f06478b5e2e9d957dbccdd9).